packages/block-editor/src/components/editor-styles/index.js

```diff
@@ -67,15 +67,18 @@ function useDarkThemeBodyClassName( styles, scope ) {
 	);
 }
 
-function EditorStyles( { styles, scope } ) {
-	const overrides = useSelect(
-		( select ) => unlock( select( blockEditorStore ) ).getStyleOverrides(),
-		[]
+function EditorStyles( { styles, scope, overrides } ) {
```

Comment:
Converted to draft while I refactor and question life choices.

This overrides prop, overrides the overrides. 

Maybe better to merge them?

Reply:
I'm only passing EditorStyles the overrides that need to override the overrides 😄 

And now, merging them with the store overrides.